moduleframework/mtf_scheduler.py

```diff
@@ -248,6 +250,45 @@ def avocado_general(self, avocado_default_args=[]):
             rc = cpe.returncode
         return rc
 
+    def _tcinfo(self, testcases, header, logs=True, description=True):
+        """
+        Parse testcases dics and print output for them in nicer format
+        Main purpose is to display docstrings of testcases for failures
+        :param testcases: dict of testcases
+        :param header: str what to print as header
+        :param logs: boolean if print logs for these testcases (default True)
+        :param description: boolean if print description = docs strings for test class + function
+        :return: None
+        """
```

Comment:
What about to add there an example of test?
E.g.
~~~
def test_some(something)
   docs: something1
   result: something2
~~~

Reply:
Yep, could be. Actually I ignore all lines contains any `:.+:` strings and print everything else as onneliner.
Probably we can improve it in future. and use some semantics in docstrings. Does that  make sesne?
For example I can imagine, that there could be both, in case there will be these marks it will use them. in case there will be just docsstring,  this string will be printed out.

Reply:
Looks better, but some digging. What `fail_reason` is printed? Something from docstring? Or from assert?
If from assert, I would add a function like

~~~
self.assert(tests, "result for fail_reason")
~~~